roles/neutron-data-network/tasks/main.yml

```diff
@@ -70,17 +70,30 @@
     group: neutron
     mode: 0755
 
-- name: modified neutron lbaas templates to workaround 2000 connection limit
-  copy:
-    src: etc/neutron/lbaas_templates/{{ item }}
-    dest: /etc/neutron/lbaas_templates/{{ item }}
+- name: template neutron lbaas haproxy to workaround 2000 connection limit
```

Comment:
Can this name be changed? The original workaround for 2000 connection limit will not be required in 3.1 release.